bothub/project/usecases/template_type/creation.py

```diff
@@ -0,0 +1,20 @@
+from .template_type_dto import TemplateTypeDTO
+from bothub.project.models import ProjectIntelligence, TemplateType
+
+class TemplateTypeCreationUseCase:
+
+    def get_repository_info_by_project_uuid(self, project_uuid):
```

Comment:
`get_repository_info_by_project_uuid` does not make clear the data returned by the method. `info` can mean any data type from the repository object. Clear names make the code easier to read, I recommend that you specify the returned data well.